control-plane/api/v1alpha1/proxydefaults_types_test.go

```diff
@@ -321,8 +383,30 @@ func TestProxyDefaults_Validate(t *testing.T) {
 		input          *ProxyDefaults
 		expectedErrMsg string
 	}{
+		"valid envoyExtension": {
+			input: &ProxyDefaults{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "global",
+				},
+				Spec: ProxyDefaultsSpec{
+					EnvoyExtensions: EnvoyExtensions{
+						EnvoyExtension{
+							Name:      "aws_request_signing",
+							Arguments: json.RawMessage(`{"AWSServiceName": "s3", "Region": "us-west-2"}`),
+							Required:  false,
+						},
+						EnvoyExtension{
+							Name:      "zipkin",
+							Arguments: json.RawMessage(`{"ClusterName": "zipkin_cluster", "Port": "9411", "CollectorEndpoint":"/api/v2/spans"}`),
+							Required:  true,
+						},
+					},
+				},
+			},
+			expectedErrMsg: "",
+		},
 		"meshgateway.mode": {
-			&ProxyDefaults{
+			input: &ProxyDefaults{
```

Comment:
You can ignore most of these, I just autogenerated the field names rather than leaving them blank